Postman Test Suite/Ed-Fi ODS-API Integration Test Suite AuthorizationTests.postman_collection.json

```diff
@@ -1,53 +1,18 @@
 {
 	"info": {
-		"_postman_id": "17588d9d-936f-4ddd-ad10-a65bda915841",
+		"_postman_id": "f10b065d-408d-4dab-b8b0-7712923d69e1",
```

Comment:
changes inside "info" can be reverted, those are values assigned by postman based on the environment/user that exported the collection